features/update/validation_replay/README.md

```diff
@@ -0,0 +1,11 @@
+# Replay does not perform update validation
```

Comment:
Should we also test the update function does run on replay?

Reply:
Maybe I'm wrong about what the harness does during replay but if the update didn't run under replay then we wouldn't see the activity and I would have expected that to get detected as mismatched history?

Reply:
No your right ignore me. Yeah the history should fail determinism if the activity didn't run